app/templates/admin/createSingleEvent.html

```diff
@@ -111,24 +111,22 @@ <h1 id="title">{{page_title}}</h1>
               <label class="form-label me-2" for="startTime"><strong>Start Time</strong></label>
               {% if isEditable %}
               <div class="input-group">
-                {% set startTime = eventData.timeStart.strftime('%H:%M') if eventData.timeStart and eventData.timeStart.strftime else eventData.timeStart%}
-                <input autocomplete="off" type="time" class="form-control timepicker" id='startTime' value="{{startTime}}" name="timeStart" aria-describedby="timeIcon1" placeholder="Pick a start time" required />
+                <input autocomplete="off" type="time" class="form-control timepicker readonly" value="{{ eventData.timeStart.strftime('%H:%M') if eventData.timeStart and eventData.timeStart.strftime else eventData.timeStart }}" name="timeStart" aria-describedby="timeIcon1" placeholder="Pick a start time" onchange="update(this)" id=startTime required />
                 <span class="input-group-text timeIcons" id="timeIcon1" hidden><i class="bi bi-clock"></i></span>
               </div>
               {% else %}
-                <p>{{eventData.timeStart.strftime('%H:%M') if eventData.timeStart }}</p>
+                <p>{{eventData.timeStart.strftime('%H:%M') if eventData.timeStart and eventData.timeStart.strftime else eventData.timeStart}}</p>
```

Comment:
We shouldn't need the formatting anymore, just output the text. Double check this assumption

Reply:
resolved